packages/lib-classifier/src/store/Classification.js

```diff
@@ -3,6 +3,7 @@ import Resource from './Resource'
 import { SingleChoiceAnnotation, MultipleChoiceAnnotation, Graph2dRangeXAnnotation } from './annotations'
 
 const ClassificationMetadata = types.model('ClassificationMetadata', {
+  feedback: types.frozen({}),
```

Comment:
I don't think it should be `types.map(types.frozen({})` because then it sends/saves `classification.metadata.feedback` as a Map to API/db?

Reply:
MST was throwing errors on #533, trying to write the rules map to a frozen object. I'll check this branch and see if it works here.

The classification code needs tests. I'll look at writing those.